app/models/register.rb

```diff
@@ -30,6 +30,9 @@ class Register < ApplicationRecord
       .count(:authority_id)
   }
   scope :available_count, -> { available.in_beta.count }
+  scope :category, ->(category_id) {
```

Comment:
we don't need this, we can use rails associations instead. See: https://github.com/openregister/registers-frontend/blob/6efb3e596f25a77ac47a4600d61553457ddf2880/app/models/register.rb#L40 if we fix this association and change it to `:category` then you can just call through Registers model for the category e.g. `Register.first.category`. This also negates the need for the  instance variable in the controller.

Reply:
Sure thing - I've changed it to this method.